gapic/generator/generator.py

```diff
@@ -274,6 +274,9 @@ def _render_template(
                 if (
                         skip_subpackages
                         and service.meta.address.subpackage != api_schema.subpackage_view
+                        or
+                        'transport' in template_name
+                        and not self._is_desired_transport(template_name, opts)
```

Comment:
Can you put parentheses around this chunk, just to make it clear that it's a cohesive unit?